x-pack/plugins/observability_solution/inventory/public/components/grouped_inventory/styles.tsx

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Could this CSS be defined directly in `/inventory_page/index.tsx`. Do we need this extra file?